src/utils/iosFix.js

```diff
@@ -11,6 +11,17 @@ export const iOSfix = () => {
     document.body.style.top = `${offset * -1}px`
     dom.addClass(document.body, swalClasses.iosfix)
     lockBodyScroll()
+    addBottomPaddingForTallPopups() // #1948
+  }
+}
+
+const addBottomPaddingForTallPopups = () => {
+  const safari = !navigator.userAgent.match(/CriOS/i)
```

Comment:
This line would also detect Firefox as Safari. Check:
```
!"Mozilla/5.0 (iPhone; CPU iPhone OS 10_15_5 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) FxiOS/26.0 Mobile/15E148 Safari/605.1.15".match(/CriOS/i)
```

If this line is intended to exclude only Chrome, that's right. But if its intended to limit to Safari, then there need a bit more work on that, like: 
https://stackoverflow.com/questions/3007480/determine-if-user-navigated-from-mobile-safari/29696509#29696509

```
var iOSSafari = /iP(ad|od|hone)/i.test(window.navigator.userAgent) && /WebKit/i.test(window.navigator.userAgent) && !(/(CriOS|FxiOS|OPiOS|mercury)/i.test(window.navigator.userAgent));
```

Reply:
Done, thanks! Now Firefox, Edge, Yandex, UCBrowser are excluded. It's a shame that we should detect Safari by excluding other browsers, but useragents are one of the biggest nonsences in webdev :)

Reply:
Yeah, totally agree with you! =)